docs/api/js/ethers-v6/examples/deposit.md

```diff
@@ -0,0 +1,87 @@
+---
+head:
+  - - meta
+    - name: "twitter:title"
+      content: JS SDK Deposit Example | zkSync Era Docs
+---
+
+# Deposit
+
+## Deposit ETH
+
+This is an example of how to deposit ETH from Ethereum network (L1) to zkSync Era network (L2):
+
+```ts
+import { Provider, types, utils, Wallet } from "zksync2-js";
+import { ethers } from "ethers";
+
+const provider = Provider.getDefaultProvider(types.Network.Goerli);
+const ethProvider = ethers.getDefaultProvider("goerli");
+const PRIVATE_KEY = process.env.PRIVATE_KEY;
+const wallet = new Wallet(PRIVATE_KEY, provider, ethProvider);
+
+async function main() {
+  console.log(`L2 balance before deposit: ${await wallet.getBalance()}`);
+  console.log(`L1 balance before deposit: ${await wallet.getBalanceL1()}`);
+
+  const tx = await wallet.deposit({
+    token: utils.ETH_ADDRESS,
+    to: await wallet.getAddress(),
+    amount: 7_000_000_000,
+    refundRecipient: await wallet.getAddress(),
+  });
+  const receipt = await tx.wait();
+  console.log(`Tx: ${receipt.hash}`);
+
+  console.log(`L2 balance after deposit: ${await wallet.getBalance()}`);
+  console.log(`L1 balance after deposit: ${await wallet.getBalanceL1()}`);
+}
+
+main()
+  .then()
+  .catch((error) => {
+    console.log(`Error: ${error}`);
+  });
+```
+
+## Deposit tokens
+
+This is an example of how to deposit tokens from Ethereum network (L1) to zkSync Era network (L2):
+
+```ts
+import { Provider, types, Wallet } from "zksync2-js";
+import { ethers } from "ethers";
+
+const provider = Provider.getDefaultProvider(types.Network.Goerli);
+const ethProvider = ethers.getDefaultProvider("goerli");
+const PRIVATE_KEY = process.env.PRIVATE_KEY;
+const wallet = new Wallet(PRIVATE_KEY, provider, ethProvider);
+
+async function main() {
+  // Crown token which can be mint for free
+  const tokenL1 = "0xc8F8cE6491227a6a2Ab92e67a64011a4Eba1C6CF";
+  const tokenL2 = await provider.l2TokenAddress(tokenL1);
+
+  console.log(`L2 balance before deposit: ${await wallet.getBalance(tokenL2)}`);
+  console.log(`L1 balance before deposit: ${await wallet.getBalanceL1(tokenL1)}`);
```

Comment:
I tested code, and It failed for me, potentially because an overflow error:

```
L2 balance before deposit: 0
L1 balance before deposit: 99999999999999999999999999990
Error: TypeError: underflow (argument="value", value=5e-13, code=INVALID_ARGUMENT, version=6.7.1)
```

My balance was 99999999999999999999999999990 with a test token where the decimal was 18 (99999999999.99999999999999999)

Reply:
Can you please provide your account address, token address so I can reproduce the error that you mentioned.

Reply:
Contract: 0xb24f060AEdEF6f64Abaaf33Dc72Bc699400681ea
Account: 0xAfD2d8d18B2B7D1C1f35869Df4ab66BAeFc9A912

Contract was something like this (but with more zero, I removed a few of them to fix the issue):

```
pragma solidity ^0.8.7;

import "node_modules/@openzeppelin/contracts/token/ERC20/ERC20.sol";

contract GLDToken is ERC20 {
    constructor() ERC20("Gold", "GLD") public {
        _mint(msg.sender, 1000000000000000000000);
    }
}
```

Reply:
```
Error: TypeError: underflow (argument="value", value=5e-13, code=INVALID_ARGUMENT, version=6.7.1)
```
The error is not related to the smart contract; it is related to the value itself because it's passed as a floating-point number (`0.0000000000005`) instead of an integer.